MREUnityRuntime/MREUnityRuntimeLib/Core/PrimitiveDefinition.cs

```diff
@@ -13,45 +8,11 @@ namespace MixedRealityExtension.Core.Types
     /// </summary>
     public enum PrimitiveShape
     {
-        /// <summary>
-        /// The primitive is a sphere with a radius of [[PrimitiveDefinition.radius]], horizontal segment count
-        /// [[PrimitiveDefinition.uSegments]], and vertical segment count [[PrimitiveDefinition.vSegments]], centered
-        /// at the origin.
-        /// </summary>
```

Comment:
Curious why you deleted these comments.

Reply:
+1

Reply:
This is a patch type, so this type corresponds exactly with the SDK-side documentation. If we have it twice, it just means they can drift out of sync.

---
In reply to: [316767564](https://github.com/microsoft/mixed-reality-extension-unity/pull/98#discussion_r316767564) [](ancestors = 316767564)